polynote-frontend/package.json

```diff
@@ -30,7 +30,6 @@
     "@types/jest": "26.0.19",
     "@types/node": "14.14.20",
     "babel-jest": "26.6.3",
-    "canvas": "2.6.1",
```

Comment:
I couldn't see any reason why this dependency was included, and it was transitively bringing in something with an important security warning that couldn't be fixed by `npm audit fix`. So I removed it (and it didn't seem to hurt)